.github/workflows/Basic.yml

```
@@ -53,20 +53,20 @@ jobs:
           components: rustfmt, clippy
 
       - name: Run cargo fmt
         uses: actions-rs/cargo@v1
         with:
           command: fmt
-          toolchain: 1.81.0
+          toolchain: 1.87.0
           args: --all -- --check
 
       - name: Run cargo clippy
         uses: actions-rs/cargo@v1
```

actionlint (1.7.7)

the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)